showcase/src/app/home/home.component.html

```diff
@@ -15,5 +15,39 @@ <h1>Angular Material components</h1>
 					<button mat-raised-button disabled>Disabled</button>
 				</div>
 			</mat-card-content>
+			<h1 class="documentation-title">Stark links</h1>
+			<div class="documentation-links">
+				<div>
+					<a href="https://stark.nbb.be/api-docs/stark-core/latest">
+						<img class="stark-core-latest">
```

Comment:
I think the class name should be different to really describe what it is for, maybe "img-stark-core-latest"?

In fact, I'm wondering if it is possible to add the image as a background of the `<a>` instead of having an `<img>` as a child element... then your HTML could be simplified as well as your styles.

The same for the other images...